### ISSUE TITLE
[Usage]: "How can I register Logits Processors in the args?"

### ISSUE THREAD

**Issue by hjlee1995** [vllm-project/vllm#16709]
### Your current environment

```text
The output of `python collect_env.py`

I'm using a Git-cloned version of vLLM to register a custom function, so I'm getting an error when trying to print the current vLLM version. However, I'm using a recent version of vLLM.
```

### How would you like to use vllm

Hello,

While serving with vLLM, I encountered cases where foreign languages were being generated, so I created a custom LogitsProcessor to block the generation of specific foreign languages.  
This works well when I use `llm.chat()` directly. However, I want to use this through OpenAI's `client.chat` interface instead.  
The issue is that OpenAI's client does not support custom `SamplingParams`.

I'm trying to register my custom LogitsProcessor or related `SamplingParams` via the `python3 -m vllm.entrypoints.openai.api_server` command.

While exploring the code, I came across an argument called `logits-processor-pattern`, which seems related to what I'm trying to achieve, but there are no usage examples, so I'm not sure how to use it.

If `logits-processor-pattern` is not intended for this purpose, I’d like to ask how I can register my custom LogitsProcessor using args instead.

Thank you for your time!

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by tonyaw** [comment]
I also need this feature to forbid some tokens. Could you please help to provide some usage example?

**Comment by tonyaw** [comment]
I figured it out:
1. Put your processor function into some path that vllm can import:
    Like to put https://github.com/vllm-project/vllm/discussions/3434 into /usr/local/lib/python3.12/dist-packages/vllm/my_logits_processor.py       
2. Start openai.api_server with logits-processor-pattern:
```bash
--logits-processor-pattern "vllm\.my_logits_processor\.bad_word_processor"
```
3. Send request with "logits-processor" in OpenAI client models kwargs:
```python
            "extra_body": {
                 "logits_processors": ["vllm.my_logits_processor.bad_word_processor"], 
             },
```
